### DISCUSSION TITLE
【求助】友善的社区是否可以帮忙写个模版

### DISCUSSION THREAD

**Discussion by chenmaoran** [zhaohongxuan/obsidian-weread-plugin#244] [Q&A]
今天刚发现这个插件，感觉很不错！但是感觉模版都比较复杂，也不太喜欢。

不会写代码，看了别人的代码也不知道怎么改。各位大神，能否帮忙写出如下的模版。在这里先感谢各位啦~

主要有几个要求：

1. 章节下有笔记或者高亮内容的才显示这个标题，否则不显示这个章节
2. 所有显示的章节，只显示最上层级的章节名称，不要显示太多章节层级
3. 同一个章节下如果有多条高亮/笔记的，按照正文中的先后顺序显
4. 生成的markdown文件不要有连续空行

```
### chapterTitle-1

> 高亮01

> 笔记02

> 高亮03

### chapterTitle-4

> 笔记05

> 高亮04
```

**Comment by zhaohongxuan** [comment]
第一点需要等到下个版本上线才能实现，现在默认都会打印章节标题。
